habitat_baselines/rl/ppo/policy.py

```diff
@@ -12,12 +12,20 @@
 
 
 class Policy(nn.Module):
```

Comment:
The policy should either take which type of network it uses as an argument or this is done via inheritance (I am more partial to an argument).  See bellow for the reason for different networks.